.github/workflows/build.yaml

```diff
@@ -6,26 +6,31 @@ on:
     tags:
       - v*
 
+env:
+  MAIN_REGISTRY: ghcr.io
+  MAIN_USERNAME: GHCR_USERNAME
+  MAIN_PASSWORD: GITHUB_TOKEN
+  MAIN_REPOSITORY: janus-idp/redhat-backstage-build
```

Comment:
Can this be `${{ github.repository }}` or does it have to be verbatim?

Reply:
Yes this seems to work, replaced it with `${{  github.repository }}`.